pkg/kubectl/cmd/create.go

```diff
@@ -97,11 +97,11 @@ func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) err
 	err = r.Visit(func(info *resource.Info) error {
 		data, err := info.Mapping.Codec.Encode(info.Object)
 		if err != nil {
-			return err
+			return fmt.Errorf("error when creating %q: %v ", info.Source, err)
```

Comment:
Well-- this is a public method, so anyone could call it, no? CheckErr isn't guaranteed to be the thing receiving the error?